feature/networkinstance/otg_tests/defaults_test/defaults_test.go

```diff
@@ -181,6 +183,16 @@ func TestDefaultAddressFamilies(t *testing.T) {
 			otgutils.WaitForARP(t, ate.OTG(), top, "IPv4")
 			otgutils.WaitForARP(t, ate.OTG(), top, "IPv6")
 
+			intfNbrPath := gnmi.OC().Interface(dutP2.Name()).Subinterface(0).Ipv6().Neighbor(atePort2.IPv6)
+			_, ok := gnmi.Watch(t, dut, intfNbrPath.LinkLayerAddress().State(), 5*time.Minute, func(val *ygnmi.Value[string]) bool {
```

Comment:
is there a reason why is this required in this test?

up to 5 minutes seems like a long time to wait, and this configuration is a novelty (not used in other tests). if there's a problem with how neighbor resolution is performed, perhaps that should be generalized.

Reply:
@LimeHat This is needed since ixia "VerifyArp" does not have method to send arp request. It is only sending during "Pushconfig" call.

We do not need this in case of physical devices where they are directly connected. 

If devices are containerized and VerifyArp does not actually send Arp to verify reachability. Its only using existing arp entry and continues.